### PR TITLE
Fix warnings for entities

### PR DESCRIPTION
**Pull request by Seb-stian**
Mostly fixing nullable reference warnings

### CODE REVIEW

**Review by Jonpro03 — DISMISSED** [review]
this.

**Comment by Seb-stian** [comment]
Seems that some conflicts popped up. Working on a fix now

Edit: done

File: Obsidian/Net/Packets/Play/CloseContainerPacket.cs [hunk 1]
```diff
@@ -13,9 +14,44 @@ public partial class CloseContainerPacket : IClientboundPacket, IServerboundPack
...
+                ActionId = 1,
+                ActionParam = 0,
+                BlockType = block.BaseId
+            });
+            await player.SendSoundAsync(SoundEffectBuilder.Create(SoundId.BlockEnderChestClose)
+                .WithSoundPosition(position.SoundPosition)
+                .Build());
+        }
+
+        player.OpenedContainer = null;
```
Comment (Tides) [inline Obsidian/Net/Packets/Play/CloseContainerPacket.cs]: Why was this reverted 🤨

Comment (Seb-stian) [reply Obsidian/Net/Packets/Play/CloseContainerPacket.cs]: Git probably messed up when merging with master

**Comment by Naamloos** [comment]
opsies conflicts

**Comment by Seb-stian** [comment]
Not again 😭

**Pull request merged**